### PR TITLE
Create app skeleton

### PR DESCRIPTION
**Pull request by guks**
1. installed Google fonts via the link tag
2. reset default browser styles with styled-normalize package
3. added global styles with GlobalStyles and ThemeProvider with some variables from styled-components
4. added react-router-dom routes for Home and Search pages
5. setup folder structure for the project (shared, components, pages)

### CODE REVIEW

File: src/shared/styles/global.styles.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from 'styled-components';
+import { normalize } from 'styled-normalize';
```
Comment (jkettmann) [inline src/shared/styles/global.styles.js:2]: Interesting. Have to look up that package :-)

File: src/shared/styles/global.styles.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const GlobalStyle = createGlobalStyle`
+  ${normalize}
+console.log('imtheme', props.theme);
```
Comment (jkettmann) [inline src/shared/styles/global.styles.js]: Please remove

Comment (jkettmann) [inline src/shared/styles/global.styles.js]: I guess this won't work by the way. You would need to do this

```
${props => console.log(...)}
```

File: src/shared/styles/theme.styles.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import PropTypes from 'prop-types';
+
+const theme = {
```
Comment (jkettmann) [inline src/shared/styles/theme.styles.js:5]: Great!

**Review by jkettmann — CHANGES_REQUESTED** [review]
Only a console log left over. Looks great otherwise

File: src/shared/styles/global.styles.js (hunk 2 above)
Comment (guks) [reply src/shared/styles/global.styles.js]: Removed thank you!

**Review by jkettmann — APPROVED** [review]
:+1:

**Pull request merged**